test/spec/modules/33acrossBidAdapter_spec.js

```diff
@@ -928,6 +944,236 @@ describe('33acrossBidAdapter:', function () {
         expect(builtServerRequests).to.deep.equal([serverRequest]);
       });
     });
+
+    context('when mediaType has video only and context is instream', function() {
+      it('builds instream request with default params', function() {
+        const bidRequests = (
+          new BidRequestsBuilder()
+            .withVideo({context: 'instream'})
+            .build()
+        );
+
+        const ttxRequest = new TtxRequestBuilder()
+          .withVideo()
+          .withProduct('instream')
+          .build();
+
+        ttxRequest.imp[0].video.placement = 1;
+        ttxRequest.imp[0].video.startdelay = 0;
+
+        const serverRequest = new ServerRequestBuilder()
+          .withData(ttxRequest)
+          .build();
+        const builtServerRequests = spec.buildRequests(bidRequests, {});
+
+        expect(builtServerRequests).to.deep.equal([serverRequest]);
+      });
+
+      it('builds instream request with params passed', function() {
+        const bidRequests = (
+          new BidRequestsBuilder()
+            .withVideo({context: 'instream', startdelay: -2})
+            .build()
+        );
+
+        const ttxRequest = new TtxRequestBuilder()
+          .withVideo({startdelay: -2, placement: 1})
+          .withProduct('instream')
+          .build();
+
+        const builtServerRequests = spec.buildRequests(bidRequests, {});
+
+        expect(JSON.parse(builtServerRequests[0].data)).to.deep.equal(ttxRequest);
```

Comment:
Changing assert to compare the built request rather than server payload `serverRequest` since the latter has `ttxRequest` stringified and therefore order matters making the tests very brittle.